pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -633,11 +638,13 @@
     <xs:complexType>
       <xs:sequence>
         <xs:element name="source" type="xs:string"/>
+        <!-- Why do we need this? Specifying "target software" is in conflict with PVcollada 2.0 being a common interchange format -->
         <xs:element name="target" type="xs:string" minOccurs="0"/>
```

Comment:
Why do we need this? Specifying "target software" is in conflict with PVcollada 2.0 being a common interchange format

Reply:
PVCollada will permit users to add additional <technique> objects, with content unique to a user's platform. <source> and <target> may be important for a receiver to know how to read those additional <technique> objects.